conda/environment_cpu.yaml

```
@@ -0,0 +1,12 @@
+name: FTorch-cpu
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - fortran-compiler
+  - cxx-compiler
+  - cmake >=3.15
+  - openmpi >=5.0.6  # For pFUnit
+  - pytorch-cpu >=2.5.1  # Force cpu-only version
+  - torchvision  # For example 2
```

```suggestion
  - torchvision  # For the ResNet example
```
because it's likely we'll change the example numberings soon and don't want to have to search around for references like this.